packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -69,6 +71,7 @@ const CompareCFDs = observer(() => {
                         </div>
                     </div>
                 </div>
+                {/* <CFDPasswordModal context={store} platform={platform} /> */}
```

Comment:
was this comment intentional?

Reply:
I was checking something.
Will remove it